_includes/k8s/custom-helm-install.md

```diff
@@ -0,0 +1,23 @@
+### Helm installation
+
+StorageOS can be installed with Helm. Helm adds versatility to the installation
+method. Default values can be overridden in the command line or you can edit
```

Comment:
`edit` -> pass a custom values file to override the default `values.yaml` values.
Since there's no more repo cloning, users can't edit `values.yaml`.
Maybe an example of how to pass a file.

Reply:
Added a tiny explanation and I make a not to expand after GA.